### PR TITLE
Debugging lesson 5.1 done

### PR DESCRIPTION
**Pull request by CodergirlVS**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -43,16 +54,17 @@ function checkGuess() {
...
 
     const remainingAttempts = maxNumberOfAttempts - attempts;
+    const guessVerbiage = remainingAttempts === 1 ? 'guess' : 'guesses';
 
     numberOfGuessesMessage.style.display = '';
-    numberOfGuessesMessage.innerHTML = `You guessed ${guess}. <br> ${remainingAttempts} guesses remaining`;
+    numberOfGuessesMessage.innerHTML = `You guessed ${guess}. <br> ${remainingAttempts} ${guessVerbiage} remaining`;
   }
 
-  if (attempts ==== maxNumberOfAttempts) {
```
Comment (rtesagg84) [inline index.js:55]: this equal operator should be three not four

**Review by rtesagg84 — COMMENTED** [review]
I do clone your PR to see the change you made. but i didn't get the results as the assignment requirement. so double check the assignment or reach me out on slack if you have any problem.
Thank you!

**Comment by CodergirlVS** [comment]
Hi,

Yes I worked on the assignment then in the same branch I made changes to
make my portfolio look as I wanted it on the website. So may be few things
won't match.

Also they said I don't need to flex on the form if I have used it other
places since I dint want to.change everything. But I did use flex for the
name and email input box.

I will catchup with you on slack. How about 6pm PST today. Will you be
there?

Thanks
Regards
Vaidwhi Sapre

**Comment by CodergirlVS** [comment]
Hi,

What is your ID in slack. Am not able to find you.

In debugging in my code it shows my 3 equal signs only as that is one of
the errors.

Why are you seeing 4 equal signs?

Can you message me on slack please

Vaidehi S or ***@***.***

**Pull request merged**